### ISSUE TITLE
Front-end display fields should not show zero for non-zero entries

### ISSUE THREAD

**Issue by michieldenhaan** [quintel/etlocal#131]
All ETLocal input elements that directly correspond to a node/edge in the graph have a scaled value as default. For some areas and inputs, this value is so small that it shows up as ``0.0`` in the ETLocal front-end. This prompts the user to believe that the value is zero, while in fact it is not. This may result in unintended energy flows when exporting a dataset to ETSource.

One example of this is Coal CHP production in the Industry tab. For most municipalities and neighbourhoods the ETLocal front-end shows zero demand, while the actual value is non-zero.
 
I don't think it's desirable to show the actual database value to the user (since a lot of defaults are scaled NL values, they typically have 10+ significant numbers), but it is important that the users known when values are non-zero.

Possible solution:
- Make number of significant digits shown in front-end dynamic; show values up to the first non-zero significant number; e.g. show ``0.0001`` rather than ``0.0`` if number is small

Supersedes: https://github.com/quintel/etlocal/pull/127

**Comment by mabijkerk** [comment]
@noracato is this still relevant? I do not know if we still use scaled values?

**Comment by noracato** [comment]
Yes, if a value is not set explicitly it defaults to the scaler I believe. Usually I believe we try to set everything, but @Charlottevm might know more about this
